src/module_2/EDA_1.md

```diff
@@ -0,0 +1,3314 @@
+     9   count_pets             325 non-null    float64
+    dtypes: float64(5), object(5)
+    memory usage: 428.2+ KB
+    None
+
+
+
+    ---------------------------------------------------------------------------
+
+    KeyError                                  Traceback (most recent call last)
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/core/indexes/base.py:3653, in Index.get_loc(self, key)
+       3652 try:
+    -> 3653     return self._engine.get_loc(casted_key)
+       3654 except KeyError as err:
+
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/_libs/index.pyx:147, in pandas._libs.index.IndexEngine.get_loc()
+
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/_libs/index.pyx:176, in pandas._libs.index.IndexEngine.get_loc()
+
+
+    File pandas/_libs/hashtable_class_helper.pxi:7080, in pandas._libs.hashtable.PyObjectHashTable.get_item()
+
+
+    File pandas/_libs/hashtable_class_helper.pxi:7088, in pandas._libs.hashtable.PyObjectHashTable.get_item()
+
+
+    KeyError: 'item_ids'
+
+    
+    The above exception was the direct cause of the following exception:
+
+
+    KeyError                                  Traceback (most recent call last)
+
+    Cell In[16], line 30
+         28 # Análisis de la tasa de abandono del carrito
+         29 abandoned_cart['user_id'] = abandoned_cart['user_id'].apply(lambda x: len(x))  # Contar artículos abandonados
+    ---> 30 sns.histplot(abandoned_cart['item_ids'], bins=20)
+         31 plt.title('Distribución de Artículos Abandonados')
+         32 plt.xlabel('Número de Artículos Abandonados')
+
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/core/frame.py:3761, in DataFrame.__getitem__(self, key)
+       3759 if self.columns.nlevels > 1:
+       3760     return self._getitem_multilevel(key)
+    -> 3761 indexer = self.columns.get_loc(key)
+       3762 if is_integer(indexer):
+       3763     indexer = [indexer]
+
+
+    File ~/.cache/pypoetry/virtualenvs/zrive-ds-O-AZ-S6U-py3.11/lib/python3.11/site-packages/pandas/core/indexes/base.py:3655, in Index.get_loc(self, key)
+       3653     return self._engine.get_loc(casted_key)
+       3654 except KeyError as err:
+    -> 3655     raise KeyError(key) from err
+       3656 except TypeError:
+       3657     # If we have a listlike key, _check_indexing_error will raise
+       3658     #  InvalidIndexError. Otherwise we fall through and re-raise
+       3659     #  the TypeError.
+       3660     self._check_indexing_error(key)
+
+
+    KeyError: 'item_ids'
+
+
+
+```python
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import missingno as msno
+print(f"Tamaño del dataset: {df.shape}")
+print(df.isnull().sum())
+
+msno.bar(df)
+#plt.show()
+print(orders.describe())
```

Comment:
You have already made some initial exploration like this above, cluster all related explorations.